tests/Pagination/PaginationPaginatorTest.php

```diff
@@ -174,4 +174,11 @@ public function testSimplePaginatorReturnsRelevantContextInformation()
             'prev_page_url' => '/?page=1', 'from' => 3, 'to' => 4, 'data' => ['item3', 'item4'],
         ], $p->toArray());
     }
+
+    public function testPaginatorGeneratesUrlsWithoutTrailingSlash()
```

Comment:
@phroggyy Ahh sorry, I didn't look at the rest of the test. Also I didn't notice that the line itself checks whether of not the path is the root `/` path anyway, so it's pretty obvious that it *is* accounting for this.